node/handlers/formatResponse.ts

```diff
@@ -5,13 +5,22 @@ export async function formatResponse(ctx: Context) {
 
   const quote = state.quote as Quote
 
+  const convertedQuote: Quote = {
+    skuId: quote.skuId,
+    listPrice: quote.listPrice*100,
+    costPrice: quote.costPrice*100,
```

Comment:
I like to document this specificities with inline comments..

Reply:
Makes sense